content/security.md

```diff
@@ -34,90 +35,91 @@ Without DNSSEC, a malicious hacker can point SimpleLogin MX record to their own
+Emails sent to & from our servers are encrypted using TLS versions 1.1, 1.2, or 1.3. Network attackers won't be able to man-in-the-middle (MITM) attack the emails being sent and received by you and your recipient this way.
 
-#### Others
+#### Additional Measures and Precautions
 
-In addition to the above standards, SimpleLogin mail servers also implement the following recommended standards:
+In addition to the measures and precautions stated above, SimpleLogin mail servers also implement the following standards:
 
 - [MTA-STS](https://www.hardenize.com/blog/mta-sts)
 - [TLS-RPT](https://www.hardenize.com/blog/smtp-tls-reporting-tls-rpt)
 - [DANE](https://en.wikipedia.org/wiki/DNS-based_Authentication_of_Named_Entities)
 
-
 ---
 
 ## Web Servers
 
-SimpleLogin currently has 2 web servers located at
+SimpleLogin currently runs two web app instances.
 
-- app.simplelogin.io (Germany datacenter) and
-- mx2.simplelogin.co (Netherlands) as failover.
+- app.simplelogin.io
+- mx2.simplelogin.co (fallback)
 
-All data is encrypted via [SSL/TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) when transmitted from our servers to your browser.
+All of the data in transit btween SimpleLogin servers and end users are encrypted via [SSL/TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security).
 
-In addition, we also implement the following measures:
+We again, also inmplment the following additional measures in conjunction with SSL/TLS encryption:
 
-- Only support TLS v1.2 and v1.3. TLS v1.3 supports [Forward Secrecy](https://en.wikipedia.org/wiki/Forward_secrecy).
-- Do not support previous versions (SSL, TLS 1.0, TLS 1.1) that are deprecated.
-- [DANE](https://tools.ietf.org/id/draft-ietf-dane-ops-02.html), [HSTS](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security), [CSP](https://en.wikipedia.org/wiki/Content_Security_Policy), [SRI](https://en.wikipedia.org/wiki/Subresource_Integrity), [Expect CT](https://tools.ietf.org/html/rfc6962), [X-Frame-Options](https://tools.ietf.org/html/rfc7034), [XSS Protection](https://en.wikipedia.org/wiki/Cross-site_scripting)
-
-We take reasonable security measures such as password protection, two-factor authentication for internal logins, and a whitelist of people who can deploy changes to the server. Our web app does not serve any 3rd party hosted JavaScript, except a script from [Paddle](http://paddle.com), our payment processor.
+- Versions we support: TLS v1.2 and v1.3. TLS v1.3 supports [Forward Secrecy](https://en.wikipedia.org/wiki/Forward_secrecy).
+- [DANE](https://tools.ietf.org/id/draft-ietf-dane-ops-02.html)
+- [HSTS](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security)
+- [CSP](https://en.wikipedia.org/wiki/Content_Security_Policy)
+- [SRI](https://en.wikipedia.org/wiki/Subresource_Integrity)
+- [Expect CT](https://tools.ietf.org/html/rfc6962)
+- [X-Frame-Options](https://tools.ietf.org/html/rfc7034)
+- [XSS Protection](https://en.wikipedia.org/wiki/Cross-site_scripting)
```

Comment:
These are unpopular technologies so I would put them on a single line :)

Reply:
XXS doesn't seem to be, nor does X Frame. And i put them in separate lines because it should be clear on what you have implemented on you site.